### PR TITLE
made local_run --healthcheck-only return 1 when healthchecks fail

### LINKED ISSUE #445: paasta local-run --healthcheck-only exits 0 even when the healthcheck never passes
I can give a log if you want, but it contains some internal information in it so I did not paste it here

### PR DESCRIPTION
**Pull request by mjksmith**
Fixes #445

### CODE REVIEW

File: paasta_tools/cli/cmds/local_run.py [hunk 1]
```diff
@@ -189,7 +189,7 @@ def simulate_healthcheck_on_service(
     :param container_id: Docker container id
     :param healthcheck_data: tuple url to healthcheck
     :param healthcheck_enabled: boolean
-    :returns: if healthcheck_enabled is true, then returns output of healthcheck, otherwise simply returns true
+    :returns: a 2-tuple of (healthcheck_passed_bool, healthcheck_output_string)
```
Comment (solarkennedy) [inline paasta_tools/cli/cmds/local_run.py]: Hehe, just "tuple" right?

File: paasta_tools/cli/cmds/local_run.py [hunk 2]
```diff
@@ -542,7 +542,7 @@ def run_docker_container(
 
         # If the service has a healthcheck, simulate it
         if healthcheck_mode:
-            status = simulate_healthcheck_on_service(
+            status, _ = simulate_healthcheck_on_service(
```
Comment (solarkennedy) [inline paasta_tools/cli/cmds/local_run.py]: Hmm. Makes me wonder why I/we bother to return the string part anyway if the simulate function handles printing it out.

**Comment by solarkennedy** [comment]
ship

File: paasta_tools/cli/cmds/local_run.py (hunk 1 above)
Comment (mjksmith) [inline paasta_tools/cli/cmds/local_run.py]: @solarkennedy no, http://mathworld.wolfram.com/n-Tuple.html

2-tuple or pair

Comment (solarkennedy) [inline paasta_tools/cli/cmds/local_run.py]: Oh man. TIL.

**Pull request merged**